### PR TITLE
Remove SSL hack

### LINKED ISSUE #138: Remove the SSL hack: all necessary certificates are contained
The hack mentioned is:

https://github.com/metanorma/packed-mn/blob/c1904865ca7e5375ecb4acbf2e5e86fe682ebb9f/bin/metanorma#L178-L186

_Originally posted by @ronaldtse in https://github.com/metanorma/ruby-packer/issues/37#issuecomment-995289618_

### PR DESCRIPTION
**Pull request by alexeymorozov**
Closes #138 

### Metanorma PR checklist

 - [ ] Breaking changes (list related PRs)
 - [ ] Documentation update required ([create task for this](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] External dependency introduced ([documentation update need](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] Gem with native library introduced

### CODE REVIEW

File: bin/metanorma [hunk 1]
```diff
@@ -176,15 +161,15 @@ if Gem.win_platform?
   # END of HACK
 
   require 'ffi-libarchive-binary' # to preload libarchive-13.dll
-end
 
-Net::HTTP.class_eval do
-  alias _use_ssl= use_ssl=
+  Net::HTTP.class_eval do
```
Comment (ronaldtse) [inline bin/metanorma:165]: @alexeymorozov can we remove this SSL hack?

Comment (alexeymorozov) [reply bin/metanorma:165]: We can after we find a solution for ocra. It's now applied only for Windows.

Comment (ronaldtse) [reply bin/metanorma:165]: Ah I see, thanks!

**Comment by ronaldtse** [comment]
@alexeymorozov can we merge this now? Thanks.

**Comment by alexeymorozov** [comment]
Yes, all linux and macos versions include updated certificates:

linux-x86_64 uses tebako
linux-aarch64 uses tebako
darwin-x86_64 uses tebako
darwin-arm64 uses ruby-packer

tebako uses ruby 2.7.5 which includes the cert
ruby-packer uses ruby 2.6.3 patched with the cert

**Comment by ronaldtse** [comment]
Thank you @alexeymorozov !

**Pull request merged**